documentation/src/main/markdown/integrations/acr/azurecontainerregistry.md

```diff
@@ -0,0 +1,105 @@
+# Azure Container Registry Scanning
+
+[solution_name] supports scanning images stored in the Azure Container Registry (ACR). Image scan results are sent to your dedicated [blackduck_product_name] instance providing vulnerability, license, and operational risk results on the open source software components identified in the ECR image.
+
+There are two ways that you can use [solution_name] to scan container images in ECR:
+
+- Using an Azure DevOps Pipeline, see [Azure DevOps (ADO) Plugin](../../integrations/azureplugin/azure.md)
+- Using [solution_name] on a local workstation
+
+## Prerequisites
+
+Azure Container Registry Scanning requires a fully configured instance of [solution_name].   
+   
+For prerequisite information refer to [Requirements and release information](../../gettingstarted/requirements.md)
+	
+## [solution_name] ACR scanning on a local workstation
+
+Before you can scan images in ACR using [solution_name], ensure that you satisfy the following requirements:
+
+- One or more container images stored in ACR. 
+	- For more information about publishing and storing images in ACR, refer to the [container registry topic about pushing images](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-docker-cli).
+- Azure CLI is installed
+- Docker is installed
+
+<note type="tip">To run [solution_name], you will need to provide login credentials for your [blackduck_product_name]
+server by adding the following arguments to the command line:
+
+* `--blackduck.url={your Black Duck server URL}`
+* `--blackduck.api.token={your Black Duck access token}`
+</note>
+
+To locally scan container images stored in ECR, follow these steps:
+
+1\. Authenticate with ACR.   
+The *az acr login* command generates an authentication token and authenticates with your registry.   
+  
+**Generate Docker Login for ECR (Linux)**   
+
+``` az acr login --name <acrName> ```  
+
+2\. Invoke [solution_name], and provide the following paramaters at a minimum.
+
+**[solution_name] - Scanning Images**   
+```
+bash <(curl -s -L https:‎ //detect.synopsys.com/detect8.sh) \
```

Comment:
Should this be detect9.sh?

Reply:
I think I started this back before 9 went out... lol